zoos/zoos-router.js

```diff
@@ -0,0 +1,203 @@
+const router = require("express").Router();
+const knex = require("knex");
+
+const knexConfig = {
+  client: "sqlite3",
+  useNullAsDefault: true,
+  connection: {
+    filename: "./data/lambda.sqlite3"
+  }
+};
+
+const db = knex(knexConfig);
+// endpoints here
+
+// ********** READ METHODS ********** //
+
+// GET zoos root
+
+router.get("/zoos", (req, res) => {
+  db("zoos")
+    .then(zoos => {
+      res.status(200).json(zoos);
+    })
+    .catch(error => {
+      res.status(500).json(error);
+    });
+});
+
+// GET bears root
+
+router.get("/bears", (req, res) => {
+  db("bears")
+    .then(bears => {
+      res.status(200).json(bears);
+    })
+    .catch(error => {
+      res.status(500).json(error);
+    });
+});
+
+// GET zoo by ID
+
+router.get("/zoos/:id", (req, res) => {
+  db("zoos")
+    .where({ id: req.params.id })
+    .then(zoos => {
+      res.status(200).json(zoos);
+    })
+    .catch(error => {
+      res.status(500).json(error);
+    });
+});
+
+// GET bear by ID
+
+router.get("/bears/:id", (req, res) => {
+  db("bears")
+    .where({ id: req.params.id })
+    .then(bears => {
+      res.status(200).json(bears);
+    })
+    .catch(error => {
+      res.status(500).json(error);
+    });
+});
+
+// ********** CREATE METHODS ********** //
+
+// POST a new zoo
+
+router.post("/zoos", (req, res) => {
+  const newZoo = req.body;
+  db("zoos")
+    .insert(newZoo)
+    .then(ids => {
+      const id = [ids];
```

Comment:
If you add more than one row to the db, then more than one id will be sent back to you. Console.log id here to be sure you have the right integer. Try passing in more than one newZoo to the db.